src/components/common/Header.js

```diff
@@ -18,8 +18,12 @@ const ChildMenu = props => {
     props.clearUsers();
     push('/');
   };
+
+  const setChild = useSelector(state => state.child.name);
+
   return (
     <Menu {...props}>
+      <h4> Hi, {setChild}!</h4>
       <Menu.Item key="1">
         <Link to="/child/dashboard">Home</Link>
```

Comment:
Nice utilization of useSelector! This is a great addition to the user experience.